packages/devtools_app/test/shared/auto_dispose_mixin_test.dart

```diff
@@ -105,19 +105,135 @@ void main() {
       disposer.addAutoDisposeListener(notifier, () {
         values.add(notifier.value);
       });
-      // ignore: invalid_use_of_protected_member
       expect(notifier.hasListeners, isTrue);
       notifier.value = 19;
       expect(values.length, equals(3));
       expect(values.last, equals(19));
       disposer.cancelListeners();
 
-      // ignore: invalid_use_of_protected_member
       expect(notifier.hasListeners, isFalse);
       notifier.value = 21;
       expect(values.length, equals(3));
       expect(values.last, equals(19));
     });
+
+    group('callOnceWhenReady', () {
+      for (bool isReady in [false, true]) {
+        group('readyWhen=$isReady', () {
```

Comment:
nit: change group name to 'isReady=$isReady'

Reply:
:facepalm: